cookbooks/swift/recipes/configs.rb

```diff
@@ -106,6 +106,13 @@
   creates "/usr/local/share/ca-certificates/extra/saio_ca.crt"
 end
 
+if node['full_reprovision'] then
+  execute "reinstall cert for certifi" do
+    cert_to_install = "/etc/ssl/private/saio.crt"
+    command "cat #{cert_to_install} >> $(python -m certifi)"
```

Comment:
I think it's weird that we "only" do this during full_reprovision

maybe it'd be clearer to break the "install cert" step into two parts